common/reviews/api/operation-graph.api.md

```diff
@@ -152,7 +152,7 @@ export class Operation implements IOperationStates {
     _executeAsync(context: IExecuteOperationContext): Promise<OperationStatus>;
     readonly groupName: string | undefined;
     lastState: IOperationState | undefined;
-    readonly name: string | undefined;
+    readonly operationName: string;
```

Comment:
This is also a breaking API change. Is this rename necessary?

Reply:
I feel this change is beneficial to ensure we are more explicit in the naming. The purpose of `name` is easily lost outside of the `operation-graph` lib context. Specifically as downstream libs use this interface to implement custom operations like the heft `TaskOperationRunner`.